src/beamlime/kafka/scipp_da00_compat.py

```diff
@@ -14,7 +14,13 @@ def scipp_to_da00(
             _to_da00_variable(signal_name, sc.values(da.data)),
             _to_da00_variable('errors', sc.stddevs(da.data)),
         ]
-    variables.extend([_to_da00_variable(name, var) for name, var in da.coords.items()])
+    variables.extend(
+        [
+            _to_da00_variable(name, var)
+            for name, var in da.coords.items()
+            if var.shape == var.values.shape  # vector3 etc. not supported currently
```

Comment:
Should we somehow log if some coordinates are dropped...? Or do we not care?

Reply:
We don't care for now, but we have to agree with ECDC on more details on the da00 results, i.e., what should be included.